pkgs/development/tools/build-managers/redo/default.nix

```diff
@@ -0,0 +1,32 @@
+{stdenv, fetchurl, unzip, perlPackages, perl }:
+
+stdenv.mkDerivation {
+  name = "redo-1.2";
+  src = fetchurl {
+    url = http://homepage.ntlworld.com/jonathan.deboynepollard/Softwares/redo-1.2.tar.bz2;
+    md5 = "3bbdd198b8eb4bdddc9ae39f506b2647";
+  };
+
+  nativeBuildInputs = [ perl /* for pod2man */ ];
+
+  unpackPhase = ''
```

Comment:
I'm assuming that the archive just dumps all the files into the current directory? If so, you can try setting `sourceRoot = "."` and use the default unpack phase instead. If you prefer to use a custom unpack phase, you should do something like `mkdir $name && tar xf $src -C $name && cd $name` instead, IMO.